src/showcase/ui-group/EffectCards.tsx

```diff
@@ -37,16 +25,13 @@ const EffectCards = () => {
           <Link href={item.link} key={item.id}>
             <div className="overflow group rounded-xl border border-white ring-1 ring-zinc-200 transition-all ease-in-out hover:cursor-pointer">
               <div className="flex h-[9rem] items-center justify-center rounded-t-xl border-b bg-gray-50 text-xs text-gray-400 transition-all ease-in-out group-hover:bg-gray-100 md:h-[12rem]">
-                {item.preview ? item.preview : null}
-                {item.image ? (
```

Comment:
why remove this?

Reply:
I added that in this pr when I was making this as an Effect. Since it is a Button now I changed it back to how it is in main.